merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/models/SimulationDatasetId.java

```diff
@@ -0,0 +1,3 @@
+package gov.nasa.jpl.aerie.merlin.server.models;
+
+public record SimulationDatasetId(long id) {}
```

Comment:
Proposing as, while I know our existing `Id`s use `long`, the valid range of this value (and our other `Id`s) is actually the `int` range.

```suggestion
public record SimulationDatasetId(int id) {}
```

Reply:
Happy to make this change, but should we break this precedent without changing all the other `Id` records?

Reply:
Yeah, this is a weird case. Given that, [I've created a ticket for the refactor](https://github.com/NASA-AMMOS/aerie/issues/999) so that the change can be done all at once.